packages/playground/src/weblets/profile_manager.vue

```diff
@@ -221,7 +221,7 @@
           <VBtn
             color="secondary"
             variant="text"
-            :disabled="!!ssh || updatingSSH || generatingSSH"
+            :disabled="!!ssh || updatingSSH || generatingSSH || (balance && balance.free < 0.01)"
```

Comment:
I believe that creating a helper function to check the balance and determine if it is sufficient or not would be beneficial in terms of reusability.

```ts

const isEnoughBalance = (balance: number) => {
    // Check the free balance and return true if it's enough for making transactions or not
    return (balance && balance.free < 0.01) ? true : false
}
```
```suggestion

            :disabled="!!ssh || updatingSSH || generatingSSH || !isEnoughBalance()"
```